assets/wire/components/ItemDetails.tsx

```diff
@@ -51,7 +52,7 @@ function ItemDetails({
     const media = getOtherMedia(item);
     const itemType = isPreformatted(item) ? 'preformatted' : 'text';
     const allMedia = getPictureList(item);
```

Comment:
imo it would be better to extend this func to do the filtering (and thus avoid that duplication), eg.:
```ts
getPictureList(item, {includeFeatured = false, includeEditor = false})
```